src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represents a tag operation
+ */
+public class Tagging {
+
+    private final ReadOnlyPerson person;
+    private final Tag tag;
+    private final String operation;
```

Comment:
@okkhoy 
Is it because the operation field is only storing values of "+" and "-" only, and String is too wide as it can store almost anything, and we want to narrow down the scope. Perhaps char would be more appropriate. Or perhaps an Enum called "Operation" could be defined, which allows us to change the data type and/or value in future.